docs/cave/goes-satellite.md

```diff
@@ -0,0 +1,308 @@
+# GOES 16/17
+
+The goesr EDEX decoder supports the ingest of GOES products coming over NOAAPort and Unidata's IDD. These include single channel imagery, derived products (Level 2b netCDF files), gridded Geostationary Lightning Mapper (GLM) products (produced by Eric Bruning at Texas Tech), CIRA created RGB specific products, and vertical temperature/moisture profiles. Using derived parameters, additional RGB and channel difference products can be loaded. The dmw EDEX decoder supports the ingest of GOES derived motion winds.
```

Comment:
I know the repo is called 'goesr', but I think for this paragraph, for human readability it should say "The GOES-R EDEX decoder...".
Also, the single channel imagery, derived products, glm, cira and profiles should have internal links to their corresponding sections below (for CIRA just link to the first product).

Reply:
I could color code/bold/quote the "goesr" - I was trying to use the actual name of the edex decoder that is being used - and similarly in the last paragraph the "dmw" EDEX decoder - that's the name of the EDEX decoder.

The paragraph below has all of the internal links (this was just how it was before....) but I can change it to be just in this first paragraph